app/src/main/kotlin/hr/asee/android/template/compose/ui/postlogin/home/contents/card/GiverReservationCard.kt

```diff
@@ -0,0 +1,122 @@
+package hr.asee.android.template.compose.ui.postlogin.home.contents.card
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import hr.asee.android.template.compose.R
+import hr.asee.android.template.compose.ui.common.service.Reservation
+import hr.asee.android.template.compose.ui.postlogin.home.HomeViewModel
+import hr.asee.android.template.compose.ui.theme.DarkGray
+import hr.asee.android.template.compose.ui.theme.Geomanist
+import java.time.format.DateTimeFormatter
+
+
+@Composable
+fun GiverReservationCard(reservation: Reservation, viewModel: HomeViewModel) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable { viewModel.onReservationClicked() },
+        backgroundColor = MaterialTheme.colors.onPrimary,
+        shape = RoundedCornerShape(15)
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(start = 15.dp, bottom = 8.dp, top = 8.dp)
+        ) {
+            Image(
+                painter = painterResource(
+                    id = if (isSystemInDarkTheme()) R.mipmap.parking_offer_dark
```

Comment:
Imas opciju da resurse za dark mode dodas pod istim imenom u folder s resursima s nastavkom `-night`, npr `mipmap-night`, pa ce Android sam povuci sliku koja je za dark mode iz tog foldera ako treba, inace uzme obicnu, to mozes primijeniti na sva mjesta gdje imas taj slucaj, ako treba pojasniti dodatno pitaj slobodno